src/lib/arrays.js

```diff
@@ -0,0 +1,67 @@
+exports.getNthElement = (index, array) => {
+  return array[index % array.length]; // remainder of index/array length equals given parameter in questions
+};
+
+exports.arrayToCSVString = array => {
+  return array.toString();
+};
+
+exports.csvStringToArray = string => {
+  return string.split(',');
+};
+
+exports.addToArray = (element, array) => {
+  array.push(element);
+};
+
+exports.addToArray2 = (element, array) => {
+  let expectedArray = array.concat(element);
```

Comment:
Can you return this directly instead of assigning to a variable and returning that?